### PR TITLE
feat(developer): Document the changes to the app ecosystem compatibility

### PR DESCRIPTION
**Pull request by nickvergessen**
### 🖼️ Screenshots

Bildschirmfoto vom 2023-05-04 08-46-04

### CODE REVIEW

**Review by DaphneMuller — APPROVED** [review]
Thank you for this Joas, I think it will help app developers a lot to keep up with the changes.

File: developer_manual/prologue/compatibility_app_ecosystem.rst [hunk 1]
```diff
@@ -16,18 +16,17 @@ Documentation procedures of changes that affect app developers
...
   - The steps should be explicitly written, so the documentation shall not rely upon links to external resources for the steps. While it is encouraged to add an external links as additional reference, it is a hard requirement that the documentation is readable and actionable without browsing to this link.
-  - The documentation has to be carefully written. The author's name will be added to the final publication so readers can reach out to the author if they have questions or if something is unclear.
-  - Timeline: the documentation is required to be handed in immediately after merging the pull request.
-
+  - The change author's name should be added to the section so readers can reach out to the author if they have questions or if something is unclear.
+  - Timeline: the documentation is required to be handed in when finalizing the pull request and should be merged timely close to the actual change.
 
-3. The documentation will then be copy-pasted into the developer documentation and on the forum for the first beta release.
+3. The documentation will then be copy-pasted into the forum for the first beta release.
```
Comment (ChristophWurst) [inline developer_manual/prologue/compatibility_app_ecosystem.rst:32]: does that still make sense? shouldn't we add a link to the forum pointing to the online documentation? Any changes would have to be kept in sync. Ideally the upgrade docs are frozen at release time but I'm sure there will be cases where the document is amended later. Would be bad if only the docs get an update, not the forum post.

File: developer_manual/prologue/compatibility_app_ecosystem.rst [hunk 2]
```diff
@@ -16,18 +16,17 @@ Documentation procedures of changes that affect app developers
 There are three procedures in place to facilitate communication to app developers about changes that affect them:
 
 
-1. The definition of done of a pull request includes documentation.
-This means the development of your pull request cannot be considered done if you introduced changes that affect app developers that you did not document.
+1. The definition of **Done** of a pull request includes documentation.
 
-.. TODO ON RELEASE: Change the link to the follow up issue
+  This means the development of your pull request cannot be considered done, if you introduced changes (additions, removals or modifications) that affect app developers that you did not document.
```
Comment (ChristophWurst) [inline developer_manual/prologue/compatibility_app_ecosystem.rst:21]:
```suggestion
  This means the development of your pull request cannot be considered done if you introduced changes (additions, removals or modifications) that affect app developers that you did not document.
```

nit

File: developer_manual/prologue/compatibility_app_ecosystem.rst (hunk 1 above)
Comment (nickvergessen) [reply developer_manual/prologue/compatibility_app_ecosystem.rst:32]: yeah, was unsure as well. But maybe we then always just create a cover post with the link to the newest page.
Copying it to the forum could have helped with search results. But fine by me

**Pull request merged**